signbank/dictionary/templates/dictionary/gloss_detail.html

```diff
@@ -402,7 +402,7 @@ <h4 class="modal-title" id="glossvideo{{glossvideo.pk}}-ModalLabel">{% blocktran
                 </tr>
                 <tr>
                     <th>{% blocktrans %}Film batch:{% endblocktrans %}</th>
-                    <td id='sign-number'>{{gloss.filmbatch}}</td>
+                    <td class="edit edit_text" id='sign-number'>{{gloss.filmbatch}}</td>
```

Comment:
I think if you set `id='filmbatch'` this will just work without the change in `update.py`?

Reply:
At least I'm sure I've done this for other attributes and it's just worked

Reply:
Thanks for the tip! Changed accordingly.